lib/set.js

```diff
@@ -59,13 +67,12 @@ Set.prototype.addMigration = function(title, up, down){
  */
 
 Set.prototype.save = function(fn){
-  var self = this
-    , json = JSON.stringify(this);
-  fs.writeFile(this.path, json, function(err){
-    if (err) return fn(err);
-
+  var self = this;
+  this.state.save({
```

Comment:
for some reason the migrations are not stored (only 'pos') .. isn't this necessary later on to calculate which new migrations appeared?